ee/localserver/server.go

```diff
@@ -308,6 +308,10 @@ func (ls *localServer) preflightCorsHandler(next http.Handler) http.Handler {
 		w.Header().Set("Access-Control-Allow-Headers",
 			"Accept, Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization")
 
+		// Some modern chrome and derivatives use Access-Control-Allow-Private-Network
+		// https://developer.chrome.com/blog/private-network-access-preflight/
```

Comment:
https://developer.chrome.com/blog/private-network-access-update/ newer docs say this is going to be deprecated -- do we want to add a note about that?

Reply:
```suggestion
		// https://developer.chrome.com/blog/private-network-access-preflight/
		// Though it's unclear if this is still needed, see https://developer.chrome.com/blog/private-network-access-update/
```

Reply:
Maybe like that?

Reply:
Works for me!